hamper/cli.py

```diff
@@ -22,6 +22,10 @@ def __init__(self, config):
         self.loader = PluginLoader(config)
 
         self.history = {}
+        if 'wpm' in config:
+            self.wpm = config['wpm']
+        else:
+            wpm = 200
```

Comment:
Nit: This could be done more compactly as `self.wpm = config.get('wpm', 200)`. Kind of weird that the default in the config and the default in the code are different though.